docs/source/references.bib

```diff
@@ -59,3 +145,60 @@ @inproceedings{Barral:2016
   booktitle={25th Intl Meshing Roundtable},
   year={2016}
 }
+
+@phdthesis{Wilson:2010,
+  title={Modelling multiple-material flows on adaptive unstructured meshes},
+  author={Wilson, Cian},
+  year={2010},
+  school={Imperial College London}
+}
+
+@article{Farhat:1998,
+  title={Torsional springs for two-dimensional dynamic unstructured fluid meshes},
+  author={Farhat, Charbel and Degand, Christoph and Koobus, Bruno and Lesoinne, Michel},
+  journal={Computer methods in applied mechanics and engineering},
+  volume={163},
+  number={1-4},
+  pages={231--245},
+  year={1998},
+  publisher={Elsevier}
+}
+
+@article{McRae:2018,
+  author={McRae, A. T. T. and Cotter, C. J. and Budd, C., J.},
+  title={Optimal-transport-based mesh adaptivity on the plane and sphere using finite elements},
+  journal={SIAM Journal on Scientific Computing},
+  year={2018},
+  volume={40},
+  number={2},
+  pages={1121--1148},
+  doi={10.1137/16M1109515}
+}
+
+@inproceedings{Park:2019,
+  title={Verification of unstructured grid adaptation components},
+  author={Park, Michael A and Balan, Aravind and Anderson, William K and Galbraith, Marshall C and Caplan, Philip and Carson, Hugh A and Michal, Todd R and Krakos, Joshua A and Kamenetskiy, Dmitry S and Loseille, Adrien and others},
+  booktitle={AIAA Scitech 2019 Forum},
+  pages={1723},
+  year={2019},
+  doi={10.2514/6.2019-1723}
+}
+
+@article{LeVeque:1996,
+  author={LeVeque, Randall J.},
+  title={{High-Resolution Conservative Algorithms for Advection in Incompressible Flow}},
+  journal={SIAM Journal on Numerical Analysis},
+  year={1996},
+  volume={33},
+  number={2},
+  pages={627--665},
+  doi={10.1137/0733033}
+}
+
```

Comment:
@acse-ej321 just so I don't need to open a new PR, could you please add the following reference as well? :) It's for the new demo in https://github.com/mesh-adaptation/goalie/pull/164

```suggestion

@article{Davies:2022,
  title={Towards automatic finite-element methods for geodynamics via Firedrake},
  author={Davies, D Rhodri and Kramer, Stephan C and Ghelichkhan, Sia and Gibson, Angus},
  journal={Geoscientific Model Development},
  volume={15},
  number={13},
  pages={5127--5166},
  year={2022},
  publisher={Copernicus Publications G{\"o}ttingen, Germany}
}

```

Reply:
Sorry, need to make more changes actually so I'll open a new PR myself :)

Reply:
How about you make a PR that would merge into the branch here (`50_clean_citations`)?